STYLEGUIDE.md

```diff
@@ -91,6 +97,28 @@ Markdown, etc).
     ```
 
 
+### Object ID Attribute: `objectId`
```

Comment:
Proposal: instead of `objectId` (which could cause confusion with 1622.2), how about `label`? To me, at least, it's intuitive that multiple objects can have the same label.